webform_civicrm.info.yml

```diff
@@ -1,7 +1,7 @@
 name: 'CiviCRM Webform'
 type: module
 description: 'CiviCRM and Webform integration'
-core: 8.x
+core_version_requirement: ^8.7.7
```

Comment:
`core` seems to be the required key?

I get this after applying the patch -

>Uncaught PHP Exception Drupal\Core\Extension\InfoParserException: "Missing required keys (core) in modules/contrib/webform_civicrm/webform_civicrm.info.yml" at /Applications/MAMP/htdocs/d8civicomposer/web/core/lib/Drupal/Core/Extension/InfoParserDynamic.php line 29

@KarinG Can you pls confirm?

Reply:
I suspect you're testing this on < 8.7.7? `core:` is required for Drupal Core versions before 8.7.7 ( they do not recognize the `core_version_requirement: key` )

So this PR may be a bit early and we can definitely wait. As we're moving to D9 and replacing more and more deprecated APIs we do need people on ^8.7.7 (pretty sure - looking at other modules).

Reply:
Ah ok. makes sense 👍